### DISCUSSION TITLE
Tax and Senate Functions

### DISCUSSION THREAD

**Discussion by IagoAndros** [Keriew/augustus#262] [Ideas]
Hi

In vanilla Caesar, the senate is just a big forum and provides a desirability bonus. You can complete the game without building a senate once after the tutorial. For such a grand building, it's a bit of a let down that it doesn't really do much. It should be the centerpiece of your city and as such provide a key bonus of integral function beyond just looking nice.

I was thinking to make the Senate something more vital by giving the Senate more functionality. Specifically I was thinking that the player would be unable collect tax until you have built a Senate - or perhaps you couldn't adjust the tax rate until a Senate is built. I think this would provide a bit more difficulty, while at the same time making your experience a bit more immersive, as in, "No taxation without representation!" - Though this mod is called Augustus, most of the game takes place during the Republic era after all!

Furthermore I was thinking, that there are two Senate types, the standard one, and the increased desirability one - would it be possible to separate these into two different buildings in their own right? The way I envisage it, the standard Senate would be your city's 'local' Senate, and the more desirable one could be a 'Provincial' Senate, which provides more tax powers - for example, taxing Barbarians? Or more optimistically, building on the idea of Senates being prerequisite for taxing citizens, create a mechanic where the standard Senate allows you to tax around it, like how reservoirs allow you to build fountains nearby, the Senate would allow Forums to be built within it's vicinity. The Provincial Senate would basically cover the entire map, and also any barbarian villages.

I know this is quite a lot to ask, but just thought tax and Finance is an often overlooked part of running your city and I think it would make late game more immersive and challenging.

Regardless, I would also just like to say - what an amazing mod! The work you guys have done is fantastic.

**Comment by Riww** [comment]
That's how I always think it works - no Senate means no taxes. Or is it different?

**Reply by IagoAndros** [reply]
Hi Riww

You don't need a Senate to raise taxes, Forums do it better and more flexibly. The first two tutorial missions have an objective where you must build a Senate to proceed but after that you can complete the game without it. The Senate building while grand and very desirable, takes up a lot of space, for only 30 employees, and only costs 400dn which isn't a great deal, and of course you only get to build one. Forums are your main source for tax collection. They're 4x4, you can build as many as you like - and you will need many of them to maximize your tax coverage. Players ought to have a forum on every standard housing block. The Senate is very desirable so you could save it for your palace blocks, but palace blocks have a tendency to get very big and cluttered already because of all the buildings you have to cram in to make sure the patricians are served with all those resources and infrastructure. You probably would be better off putting in a couple of forums rather than a Senate for that function. This is the thing - that example is using the Senate at it's most functional and still, the forum is superior. I want the Senate to be the center of your city, a vital building that befits it's high desirability, size and 'uniqueness' (given you can only build one). It's not even a great sink for employees. Academies are 30 workers also, take up less space, contribute to your culture rating as well as providing your citizens with higher education (though it's only required for patrician households). Hospitals contribute to your overall citizen's health and have 30 employees.

Making Senates prerequisite would mean that it has a central function, and players would have to think about where it was going to be built, plus also providing a hurdle to getting your tax base set up. I think this would add a bit of depth to the game, while at the same time making the Senate a more meaningful building.

**Comment by Riww** [comment]
The point is - you can play the game as if the Senate is necessary to collect taxes. I always thought it is the case, since the tutorial clearly suggest it, and played every mission like it is the case. 
Gameplay wise, IMO Augustus could make an option for this. At least some options would be there to make the game harder, not easier, to compensate for the many options which make the game easier compared to vanilla.

**Reply by crudelios** [reply]
I think we could make Senate a necessary building to be able to charge taxes to patricians.

So forums would always work for plebians, but in order to be able to tax patricians good you'd need to build a senate. After it's built, then patricians can also be taxed by placing forums nearby.

Edit: expanding on the idea, maybe instead of not being able to tax at all, you would be able to tax as if the building was, at most, a large insulae.

Grand insulaes and villas would require a senate to unlock the extra tax revenue.

Why start with grand insulaes and not small villas? Because I think grand insulaes are the pinnacle of pleb housing and therefore should require extra effort to attain/get its advantages.
